### ISSUE TITLE
ewfmount not releaseing with umount

### ISSUE THREAD

**Issue by slippery60** [libyal/libewf#50]
I mounted a small multi-partition .E01 image to /media/mntpt, 
used 
kpartx -a to create the /dev/mapper/loop0p1. 
then 
mount /dev/mapper/loop0p1 -o ro  /media/mntpt1

viewed two graphics files

umount /dev/mapper/loop0p1
kpartx -d /dev/mapper/loop0p1

then tried to umount /media/mntpt 
and got the error:
"umount: /media/mntpt: target is busy"

**Comment by joachimmetz** [comment]
Check with lsof what still has open files on `/media/mntpt `

**Comment by slippery60** [comment]
There were no files associated with lsof. It was one of the first things I did. 

I think I found my issue, though.  
I  re-performed all of the steps listed above. The issue is with the command 
"kpartx -d /dev/mapper/loop0p1" 

The initial command to create the "/dev/mapper/loop0p1" was
"kpartx -a /media/mntpt/ewf1"

yesterday I ran: 
        kpartx -d /dev/mapper/loop9p1

today I did:
        kpartx -d /media/mntpt/ewf1 

Everything works fine. (6 times now)

I reviewed my history, and I did run 
        kpartx -d /media/mntpt/ewf1  (the correct command)

But not until after I tried the incorrect command. I'm speculating, the incorrect kpartx command removed the node (or pieces of it), but didn't release everything it should have, or needed to.   

I re-ran the events yesterday with using the command
"kpartx -d /dev/mapper/loop0p1" results in the error I had yesterday.

it is obviously a "picnic".
